src/server/profiles/observation/observation.config.js

```diff
@@ -42,6 +42,18 @@ let routes = [
 		scopes: read_only_scopes,
 		controller: controller.searchById
 	},
+	{
```

Comment:
I believe this needs to go before searchById.  It might not actually hit this route since it matches the previous route.  I could be wrong though.

Reply:
Ok